runelite-client/src/main/java/net/runelite/client/plugins/playerindicators/PlayerIndicatorsPlugin.java

```diff
@@ -482,42 +482,60 @@ private void updateConfig()
 		if (this.highlightOwnPlayer)
 		{
 			relationColorHashMap.put(PlayerRelation.SELF, config.getOwnPlayerColor());
-			locationHashMap.put(PlayerRelation.SELF, EnumSet.copyOf(config.selfIndicatorModes()).toArray());
+			if (config.selfIndicatorModes() != null)
+			{
+				locationHashMap.put(PlayerRelation.SELF, EnumSet.copyOf(config.selfIndicatorModes()).toArray());
+			}
 		}
 
 		this.highlightFriends = config.highlightFriends();
 		if (this.highlightFriends)
 		{
 			relationColorHashMap.put(PlayerRelation.FRIEND, config.getFriendColor());
-			locationHashMap.put(PlayerRelation.FRIEND, config.friendIndicatorMode().toArray());
+			if (config.friendIndicatorMode() != null)
+			{
+				locationHashMap.put(PlayerRelation.FRIEND, config.friendIndicatorMode().toArray());
+			}
 		}
 
 		this.highlightClan = config.highlightClan();
 		if (this.highlightClan)
 		{
 			relationColorHashMap.put(PlayerRelation.CLAN, config.getClanColor());
-			locationHashMap.put(PlayerRelation.CLAN, config.clanIndicatorModes().toArray());
+			if (config.clanIndicatorModes() != null)
+			{
+				locationHashMap.put(PlayerRelation.CLAN, config.clanIndicatorModes().toArray());
+			}
```

Comment:
if (this.clanIndicatorModes() != null)
			{
				locationHashMap.put(PlayerRelation.CLAN, this.clanIndicatorModes.toArray());
			}